app/js/lib/foxbox/services/motion-sensor.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+import BaseService from './base';
+
+const TYPE = 'motion-sensor';
+
+export default class MotionSensorService extends BaseService {
+  constructor(props, config) {
+    super(props, config);
+    Object.seal(this);
+  }
+
+  get type() {
+    return TYPE;
+  }
```

Comment:
We need basic setter/getter here even if we don't use them right now, we'll need them in the future.